CHANGELOG.md

```diff
@@ -3,6 +3,19 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
+## 2.0.4 - 2017-11-2
+
+### Added
+- Host header and version to signing (#340)
+- Support for block volume fast clones (#347)
+
```

Comment:
Add:
### Fixed
- Examples of "oci_core_images" data source now filter on "display_name" to accomodate changes to available images (#342 and #345)